src/packages/src/xroad/ubuntu/generic/xroad-confproxy.postinst

```diff
@@ -1,43 +1,114 @@
 #!/bin/bash
+set -e
 . /usr/share/debconf/confmodule
 
+handle_error() {
+  ERR=$1
+  db_reset xroad-common/cert-generation-retry
+  db_reset xroad-common/cert-generation-error
+  db_subst xroad-common/cert-generation-error ERR "$(printf %s "$ERR" | debconf-escape -e)"
+  db_beginblock
+  db_input critical xroad-common/cert-generation-error || true
+  db_input critical xroad-common/cert-generation-retry || true
+  db_endblock
+  db_go
+
+  db_get xroad-common/cert-generation-retry
+  if [ "$RET" = false ]; then
+    exit 1
+  fi
+
+  echo "Retrying certificate generation"
+}
+
+create_conf_proxy_certificate() {
```

Comment:
A minor thing, but having cert generation in a separate script file and just calling it during pacakge installation might allow quick re-generation whenever it is needed.

Reply:
Re-generation can be invoked with dpkg-reconfigure. Also, the bulk of certificate generation is already in generate_certificate.sh script that can be invoked manually. 
I think I'm going to leave it as is it is for now.